touk-bash-gerrit.sh

```diff
@@ -0,0 +1,57 @@
+#!/bin/bash
+
+# Licensed to the Apache Software Foundation (ASF) under one
+# or more contributor license agreements.  See the NOTICE file
+# distributed with this work for additional information
+# regarding copyright ownership.  The ASF licenses this file
+# to you under the Apache License, Version 2.0 (the
+# "License"); you may not use this file except in compliance
+# with the License.  You may obtain a copy of the License at
+#
+#   http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing,
+# software distributed under the License is distributed on an
+# "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+# KIND, either express or implied.  See the License for the
+# specific language governing permissions and limitations
+# under the License.
+
+# This is a part of touk-bash.
+# Developed by TouK.
+
+# Includes
+. "$TOUK_BASH_HOME/touk-bash-core.sh"
+
+REVIEW_SUFFIX="rev"
```

Comment:
This is not used here so I guess it does not belong here.

Reply:
I need it in both review and cleanup scripts.

Reply:
Made it a bit more clear with a function